ucfcs/GrowthPlus
# Ruth/language settings

Addresses issues #43 and #75 

Added language preference settings which update through onResume functions().
Test by:
- Deleting and reinstalling app (the first time you open the launch screen should say Arrose+)
- Navigate to language settings screen and change language to English
- Close app and reopen, launch screen should say Growth+
- Go to different screens and change language back and forth between English/French

Also reformatted grids for landing page and parent portal. 
Styled login and sign up pages. 
Added some images for unit lessons.
